index.js

```diff
@@ -133,9 +132,10 @@ else if (program.hub) {
     console.log("------ Hub enumerate devices for %j".header, program.hub);
 
     var fileName = helpers.createOnboardingFileName(program.hub);
-    helpers.readFile(fileName, "Please complete onboarding -o").then(data => { 
-        var deviceInfo = JSON.parse(data).authInfo;
-        translatorCli.getProperty(program.hub, deviceInfo, 'getPlatforms').then(info => {
+    helpers.readFile(fileName, "Please complete onboarding -o").then(data => {
+        var configInfo = JSON.parse(data);
+        var deviceInfo = configInfo.authInfo;
+        translatorCli.getProperty(configInfo.translatorPackageName, deviceInfo, 'getPlatforms').then(info => {
             helpers.logObject(info);
```

Comment:
There's a lot of legacy logging code in here that should be cleaned up.  I'd prefer to do that as it's own change.

---
In reply to: [103610610](https://github.com/openT2T/opent2t-cli/pull/24#discussion_r103610610) [](ancestors = 103610610)